build_config/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.2.5-pre
```

Comment:
`-pre`? What happened to `-dev`?

Reply:
Lol so @natebosch gave some pushback on this earlier. I just started doing that for unreleased versions to make it more clear. If you all think its too weird I can stop though.